Weather monitoring and reporting service/Bots/Configurations/BotsConfigurations.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Newtonsoft.Json.Linq;
+
+namespace weather_monitoring_and_reporting_service;
+public class BotsConfigurations
+{
+    private List<IWeatherBotService> _botsServices { get; set; } = new List<IWeatherBotService>();
+    public List<IWeatherBotService> LoadBotConfiguration()
+    {
+        var baseDirectory = AppDomain.CurrentDomain.BaseDirectory;
+        var filePath = Path.Combine(baseDirectory, @"..\..\..\botsConfig.json");
+        var fullPath = Path.GetFullPath(filePath);
+
+        if (!File.Exists(fullPath))
+        {
+            Console.WriteLine(ErrorMessages.FileNotFount);
+            return null;
+        }
+
+        try
+        {
+            var configJson = File.ReadAllText(fullPath);
+            var jsonObject = JObject.Parse(configJson);
```

Comment:
why do you used newton JSON here, you already used Json serializer in other places try to be consistent